tests/src/rules/default.js

```diff
@@ -1,305 +1,229 @@
           code: `import foobar from "./typescript-export-assign-mixed"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
         }) : [],
         test({
           code: `import foobar from "./typescript-export-assign-default-reexport"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
         }),
         test({
           code: `import React from "./typescript-export-assign-default-namespace"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
-          parserOptions: {
-            tsconfigRootDir: path.resolve(__dirname, '../../files/typescript-export-assign-default-namespace/'),
-          },
         }),
         test({
           code: `import Foo from "./typescript-export-as-default-namespace"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
-          parserOptions: {
-            tsconfigRootDir: path.resolve(__dirname, '../../files/typescript-export-as-default-namespace/'),
-          },
         }),
         test({
           code: `import Foo from "./typescript-export-react-test-renderer"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
-          parserOptions: {
-            tsconfigRootDir: path.resolve(__dirname, '../../files/typescript-export-react-test-renderer/'),
-          },
         }),
         test({
           code: `import Foo from "./typescript-extended-config"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
-          parserOptions: {
-            tsconfigRootDir: path.resolve(__dirname, '../../files/typescript-extended-config/'),
-          },
         }),
         test({
           code: `import foobar from "./typescript-export-assign-property"`,
-          parser,
           settings: {
-            'import/parsers': { [parser]: ['.ts'] },
             'import/resolver': { 'eslint-import-resolver-typescript': true },
           },
         }),
-      ),
 
-      invalid: [
-        test({
+    // es2022: Arbitrary module namespace identifier names
+    testVersion('>= 8.7', () => ({
+      code: 'export { "default" as bar } from "./bar"',
+      parserOptions: {
+        ecmaVersion: 2022,
+      },
+    })),
+
+    ...SYNTAX_CASES,
+   ]),
```

Comment:
```suggestion
     SYNTAX_CASES,
   )),
```
the spread is no longer needed if we use concat

Reply:
okay.